notebooks/stage/keyword_analysis.ipynb

```diff
@@ -24,6 +24,8 @@
     "import re\n",
```

Comment:
<p>FYI this and the later 4 cells would result in empty outputs, since <code>clean.Chunk == datetime.date(2020, 11, 1)</code>  will not be fulfilled in automation. Maybe we should put it in <code>if not RUN_IN_AUTOMATION</code>   ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>yes, good catch. I'll change this to use the most recent "Chunk", so it should always work. </p>

Reply:
After looking at it, I decided to use your solution and use a `if not RUN_IN_AUTOMATION` statement instead of using the most recent chunk since that doesn't really make sense to do in the automation case.